src/module/helpers/setResourceObjectsForModule.js

```diff
@@ -1,4 +1,22 @@
-import { hasOwn } from '../../shared/utils'
+/**
+ * Make sure we call the correct mutation depending on whether we're mutating
+ * our own state or that of an included module.
+ *
+ * @param {string} currentModule
+ * @param {string} destinationModule
+ * @returns {string}
+ */
+function compileMutation (currentModule, destinationModule) {
```

Comment:
I think the naming is a bit odd. I wouldn't expect that a generic `compileMutation` function will only return `set` mutations

Reply:
Even in the context of this file? I don't think of non-exported functions in a global context so the naming fit for me. Do you have a suggestion for a better name?